src/Docker.DotNet/Models/ServiceFilter.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+
+namespace Docker.DotNet.Models
+{
+    public class ServicesListParameters
+    {
+        public ServiceFilter Filters { get; set; }

```

Comment:
I was hoping that these would use `[QueryStringParameter("filters", false, typeof(MapQueryStringConverter))]` and you would enlighten `MapQueryStringConverter` to process these so we could avoid adding `FilterServiceQueryString`.